examples/build/board_build_example.py

```diff
@@ -10,19 +10,7 @@
 import rhea.build as build 
 from rhea.build.boards import get_board
 from rhea.build.boards import get_all_board_names
-#from blink import blinky
-
-# some boards don't have LEDs but they have IO that
-# can drive LEDs.  The following maps the led port 
-# to a set of pins for boards without an led port.
-led_port_pin_map = {
-    'xula':  dict(name='led', pins=(36, 37, 50, 52)),
-    'xula2': dict(name='led', pins=('R7', 'R15', 'R16', 'M15',)),
-    'pone': dict(name='led', pins=(18, 23, 26, 33)),
-    # the de0-cv led port is named "ledr", for the generic testing
-    # add the led port
-    'de0cv': dict(name='led', pins=('AA2', 'AA1', 'W2', 'Y3',))
-}
+from rhea.build.boards import led_port_pin_map
```

Comment:
On 05/11/15 20:25, Christopher Felton wrote:
> In examples/build/board_build_example.py
> <https://github.com/cfelton/rhea/pull/4#discussion_r44056522>:
> 
>> @@ -10,19 +10,7 @@
>>  import rhea.build as build 
>>  from rhea.build.boards import get_board
>>  from rhea.build.boards import get_all_board_names
>> -#from blink import blinky
>> -
>> -# some boards don't have LEDs but they have IO that
>> -# can drive LEDs.  The following maps the led port 
>> -# to a set of pins for boards without an led port.
>> -led_port_pin_map = {
>> -    'xula':  dict(name='led', pins=(36, 37, 50, 52)),
>> -    'xula2': dict(name='led', pins=('R7', 'R15', 'R16', 'M15',)),
>> -    'pone': dict(name='led', pins=(18, 23, 26, 33)),
>> -    # the de0-cv led port is named "ledr", for the generic testing
>> -    # add the led port
>> -    'de0cv': dict(name='led', pins=('AA2', 'AA1', 'W2', 'Y3',))
>> -}
>> +from rhea.build.boards import led_port_pin_map
> 
> Didn't you move this to test_boards.py? Shouldn't this fail?
> 

This is the removement, not sure I understand the question!

Rob